### DISCUSSION TITLE
Meta "Database Version" endpoint

### DISCUSSION THREAD

**Discussion by natanmaia95** [PokeAPI/pokeapi#1121] [Ideas]
Unless I completely missed something, I couldn't find an endpoint about the database / api itself, specifically to know which subversion of v2 it is currently on.
My main idea was to save some of the data (like pokemon and ability lists, low res images) in a local cache for an app, and when checking the cache also checking if the version is the same as newest to update the cache with new data. Would also be nice to differentiate what server is being accessed, between the website, localhosts, other forks etc. Real meta stuff.

**Comment by Naramsim** [comment]
Maybe we could add that info into some http headers!
